types/jsreport-phantom-pdf/index.d.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types='jsreport-core' />
+
+declare namespace JsReport {
+  const enum RecipeType {
+    PhantomPdf = 'phantom-pdf'
+  }
+
+  interface Margin {
+    left: number | string;
+    right: number | string;
+    top: number | string;
+    bottom: number | string;
+  }
+
+  interface Phantom {
+    margin: string | Margin;
+    header: string;
+    footer: string;
+    width: string;
+    height: string;
+    headerHeight: string;
+    footerHeight: string;
+    format: string;
+    orientation: 'portrait' | 'landscape';
+    blockJavaScript: 'true' | any;
+    resourceTimeout: number;
+    waitForJS: 'true' | 'false';
+    fitToPage: 'true' | 'false';
+    customPhantomJS: 'true' | 'false';
```

Comment:
The boolean types should be defined as boolean. Or as 'true' | 'false' | boolean.
See the meta types declared in jsreport [here](https://github.com/jsreport/jsreport-phantom-pdf/blob/master/lib/phantom.js#L42).

Reply:
I define this just because i saw [this](https://github.com/jsreport/jsreport-phantom-pdf/blob/master/lib/phantom.js#L152)

Reply:
I think if it is defined as `boolean`, may be cause an excaption?

Reply:
`JSON.parse(true)` returns true. This code is there to assure both `"true"` and `true` are resolved as boolean. The reason for this is that the html form submits from studio are sending urlencoded values which are strings, not booleans. We make sure both are resolved as correct boolean.  Maybe jsreport-core types doesn't need to respect this and strictly demand boolean. Or at least `'true' | 'false' | booolean`.

Reply:
OK

Reply:
Do you know how to define this?
```ts
blockJavaScript: 'true' | ???;
```
I could not figure out from the source.

Reply:
The `blockJavaScript: 'true' | 'false' | boolean` should work.
However, with deeper thinking. Please keep there just a boolean. We may remove support for passing strings in future. And it is better that typescript users are already instructed to send boolean.

Reply:
OK